src/arkode/arkode_io.c

```diff
@@ -1654,7 +1727,7 @@ int ARKodeSetSafetyFactor(void* arkode_mem, sunrealtype safety)
   }
 
   /* check for allowable parameters */
-  if (safety >= ONE)
+  if (safety > ONE)
```

Comment:
Why is a safety factor of 1 not allowed?

Reply:
This is checking if an error should be raised. So before this change, a safety factor of 1 was not allow. This branch now allows a safety factor or 1. This should be allowed since there are many standard heuristics that use a safety factor of 1 with a bias>1.

Reply:
Ah, yes. If it's not already, update the docs for this function with the new allowable range and add a `versionchanged` entry to the function